website/content/docs/connect/transparent-proxy-k8s.mdx

```diff
@@ -0,0 +1,179 @@
+---
```

Comment:
This is a copy of transparent-proxy.mdx, since I wasn't able to have 2 pages on docs-nav-data.json point to the same mdx file for some reason. Posting here in case someone has an idea for a way to not just copy this file :)

Reply:
I've added href field instead, that way we can just use one page!